components/VideoPlayer.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <v-container fill-height fluid class="pa-0">
+    <div ref="videoContainer">
+      <video ref="videoPlayer" :poster="poster" autoplay></video>
+    </div>
+  </v-container>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { stringify } from 'qs';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import shaka from 'shaka-player/dist/shaka-player.ui';
```

Comment:
I have, but unfortunately that repository is severely out of date. We also prefer to stick with the upstream versions of packages.

The Shaka team does intend to ship typings very soon, however, so this will be fixed when the next version of Shaka Player is released (but that's a guess based on recent activity on their repository)